modules/cassandra/cassandra.go

```diff
@@ -14,28 +16,35 @@ import (
 )
 
 const (
-	port = nat.Port("9042/tcp")
+	port       = nat.Port("9042/tcp")
+	securePort = nat.Port("9142/tcp") // Common port for SSL/TLS connections
```

Comment:
question: do we need to two ports or can it just be one which is either SSL or not?